org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/PrivateEmptyNoArgConstructorFilter.java

```diff
@@ -37,7 +37,8 @@ public void filter(final MethodNode methodNode,
 		private boolean match(final MethodNode methodNode,
 				final String superClassName) {
 			firstIsALoad0(methodNode);
-			nextIsInvokeSuper(superClassName, CONSTRUCTOR_DESC);
+			nextIsInvoke(Opcodes.INVOKESPECIAL, superClassName, "<init>",
```

Comment:
We have a contant `CONSTRUCTOR_NAME` in this filter. Either we use it consistently or we remove it at all.

Reply:
fixed